src/main/java/pl/niewiemmichal/underhiseye/web/endpoints/AddressEndpoint.java

```diff
@@ -0,0 +1,49 @@
+package pl.niewiemmichal.underhiseye.web.endpoints;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import pl.niewiemmichal.underhiseye.commons.exceptions.*;
+import pl.niewiemmichal.underhiseye.model.Address;
+import pl.niewiemmichal.underhiseye.repository.AddressRepository;
+
+import java.util.List;
+
+@Controller
+@RequestMapping ("/addresses")
+@ResponseBody
```

Comment:
@RestController is a specialized version of the controller. It includes the @Controller and @ResponseBody annotations and as a result, simplifies the controller implementation